storage/migrations/01_oasis_3_consensus.up.sql

```diff
@@ -51,24 +51,24 @@ CREATE TABLE oasis_3.transactions
   code    UINT31,  -- From https://github.com/oasisprotocol/oasis-core/blob/f95186e3f15ec64bdd36493cde90be359bd17da8/go/consensus/api/transaction/results/results.go#L20-L20
   message TEXT,
 
-  -- We require a composite primary key since duplicate transactions can
+  -- We require a composite primary key since duplicate transactions (with identical hashes) can
   -- be included within blocks for this chain.
-  PRIMARY KEY (block, txn_hash, txn_index)
```

Comment:
👍 I do wonder if we'll have to index this later 🤔, but probably better we do those more systematically.